.goreleaser.yaml

```
@@ -17,6 +17,7 @@ builds:
     binary: celestia-appd
     env:
       - SDKPath={{ "github.com/cosmos/cosmos-sdk/version" }}
+      - CGO_ENABLED=0
```

added this to also fix ledger support for macs for the prebuilt binaries

---

I expect we need to set 

```suggestion
      - CGO_ENABLED=1
```

because if CGO is disabled, Ledger won't work

---

Nvm this won't work https://github.com/celestiaorg/celestia-app/pull/2730